repository/src/main/java/org/apache/atlas/repository/store/graph/AtlasRelationshipStore.java

```diff
@@ -35,21 +37,21 @@ public interface AtlasRelationshipStore {
      * @param relationship relationship instance definition
      * @return AtlasRelationship d
      */
-    AtlasRelationship create(AtlasRelationship relationship) throws AtlasBaseException;
+    AtlasRelationship create(AtlasRelationship relationship, boolean notify) throws AtlasBaseException;
```

Comment:
we are passing true at all places, do we need this?

Reply:
This is to specify if we want to send notification in this method itself, because these methods can be called from many other flows where we want to send notifications from the calling method in batches.